app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+  def new; end
+
+  def show
+    @user = User.find_by id: params[:id]
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+
+    if @user.save
+      flash[:success] = t(".welcome_to_pet_spa_house")
+      redirect_to @user
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:name, :email, :password,
```

Comment:
viêt 1 hằng số trong model User
USER_PARAMS = [:name, :email, :password, :password_confirmation].freeze
rồi gọi ra ở đây kiểu dạng 
```suggestion
    params.require(:user).permit User::USER_PARAMS
```